oneandone/src/main/java/org/apache/jclouds/oneandone/rest/compute/OneandoneComputeServiceAdapter.java

```diff
@@ -259,19 +289,25 @@ public void destroyNode(String id) {
 
    @Override
    public void rebootNode(String id) {
+      Server srv = api.serverApi().get(id);
+      if (srv.status().state() == Types.ServerState.DEPLOYING) {
+         return;
+      }
+      waitServerUntilAvailable.apply(getNode(id));
+      api.serverApi().updateStatus(id, Server.UpdateStatus.create(Types.ServerAction.REBOOT, Types.ServerActionMethod.HARDWARE, false, null));
       waitServerUntilAvailable.apply(getNode(id));
-      api.serverApi().updateStatus(id, Server.UpdateStatus.create(Types.ServerAction.REBOOT, Types.ServerActionMethod.HARDWARE));
    }
 
    @Override
    public void resumeNode(String id) {
-      api.serverApi().updateStatus(id, Server.UpdateStatus.create(Types.ServerAction.POWER_ON, Types.ServerActionMethod.HARDWARE));
+      api.serverApi().updateStatus(id, Server.UpdateStatus.create(Types.ServerAction.POWER_ON, Types.ServerActionMethod.HARDWARE, false, null));
```

Comment:
what happens if the node is already started?

Reply:
nothing happens